bundles/binding/org.openhab.binding.isy/build.properties

```diff
@@ -0,0 +1,7 @@
+source.. = src/main/java/,\
+           src/main/resources/
+bin.includes = META-INF/,\
+               .,\
+               OSGI-INF/,\
+               lib/isy_inst.jar
```

Comment:
I have emailed a request to the ISY team regarding this question.   I'll report back when I get an answer.